### DISCUSSION TITLE
如何在 React Native 中声明全局类型

### DISCUSSION THREAD

**Discussion by giscus** [youngjunning/blog.zisheng.pro#119] [Announcements]
# 如何在 React Native 中声明全局类型

本文讲解了如何在 React Native 中声明全局类型

https://youngjuning.js.org/d46ebc530d48/?highlight=%E5%85%A8%E5%B1%80%E7%B1%BB%E5%9E%8B

**Comment by youngjunning** [comment]
给 window 扩展类型，这么写

```ts
export {};

declare global {
  interface Window {
    foo: string;
  }
}
```
